app/tests/spec/views/complete_reset_password.js

```diff
@@ -273,7 +271,7 @@ function (chai, sinon, p, AuthErrors, Metrics, FxaClient, InterTabChannel,
       });
 
       it('non-direct-access signs the user in and redirects to `/reset_password_complete` if broker does not say halt', function () {
-        var account;
+        var account; //eslint-disable-line no-unused-vars
```

Comment:
Oh right, it returns a Promise. You'll have to put something like this as the last assertion in the block instead:

```js
return user.setSignedInAccount.returnValues[0].then(function (returnValue) {
  assert.isTrue(broker.afterCompleteResetPassword.calledWith(returnValue));
});
```

The return statement ensures that if the assertion fails and causes the promise to throw, the test will fail.